### PR TITLE
add another properties of FileStream

### PR DESCRIPTION
**Pull request by Rwing**
Please fill in this template.

- [ ] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not provide its own types, and you can not add them.
- [ ] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, and `strictNullChecks` set to `true`.

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

If removing a declaration:
- [ ] If a package was never on DefinitelyTyped, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

**Comment by Rwing** [comment]
@shepherdwind

File: types/egg/index.d.ts [hunk 1]
```diff
@@ -535,6 +535,20 @@ export interface Application extends EggApplication {
 
 interface FileStream extends Readable { // tslint:disable-line
   fields: any;
+
```
Comment (shepherdwind) [inline types/egg/index.d.ts]: tslint fail

```
ERROR: 538:1  no-trailing-whitespace  trailing whitespace
ERROR: 540:1  no-trailing-whitespace  trailing whitespace
ERROR: 542:1  no-trailing-whitespace  trailing whitespace
ERROR: 544:1  no-trailing-whitespace  trailing whitespace
ERROR: 546:1  no-trailing-whitespace  trailing whitespace
ERROR: 548:1  no-trailing-whitespace  trailing whitespace
ERROR: 550:1  no-trailing-whitespace  trailing whitespace
```

Comment (shepherdwind) [inline types/egg/index.d.ts]: @Rwing The egg declaration file is bundling with npm package now. You can just send pr to the [egg](https://github.com/eggjs/egg) repo.

Comment (Rwing) [inline types/egg/index.d.ts]: ok, got it

**Comment by sandersn** [comment]
@Rwing the reason the at-mention failed is that it's @shepherdwind not @sheperdwind.  You can change it in this PR if you want but you don't have to.

**Comment by shepherdwind** [comment]
@sandersn the egg type declaration file had move to npm package now. Should I send mr to remove the package or add egg to notNeededPackages.json.

**Comment by sandersn** [comment]
Yes, please remove it. Looks like we have [an `npm run` task that does this](https://github.com/DefinitelyTyped/DefinitelyTyped#removing-a-package).

Should I close this PR?

**Comment by shepherdwind** [comment]
Yes, This pr can close now, this change has commit to egg now.

**Pull request closed without merging**